app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="app_name" translatable="false">AutoDark</string>
+    <string name="app_about">Über AutoDark</string>
+    <string name="app_confirm">OK</string>
+    <string name="app_later">Später erinnern</string>
+    <string name="app_ranko_email" translatable="false">ranko0p@outlook.com</string>
+    <string name="app_github_page" translatable="false">https://github.com/0ranko0P/AutoDark</string>
```

Comment:
"translatable=false" means this string is untranslatable, you don't need to copy these strings.

Reply:
Ah, I just didn't translate them 😅